contents/newsletter/small-teams.md

```diff
@@ -0,0 +1,96 @@
+---
+title: The secret to shipping really fast
```

Comment:
We already have "How to design your company for speed" so we need a different headline for this.

Some options:
- The magic of one pizza teams (reference to an Amazon thing, we could work this into the introduction)
- Why and how we built our startup around small teams
- How small teams work and why they win

Reply:
Yup, will have a think and update.